docs/charts/waterfall_chart.ipynb

```diff
@@ -190,7 +226,18 @@
    "metadata": {},
    "source": [
     "### Actions to take as a result of the chart\n",
-    "\n"
+    "\n",
+    "This chart is useful for spot checking pairs of records to see if the Splink model is behaving as expected.\n",
+    "\n",
+    "If a pair of records look like the are incorrectly being assigned as a match/non-match, it is a sign that the Splink model is not working optimally. If this is the case, it is worth revisiting the model training step. Some common scenarios include:\n",
+    "\n",
+    "- If a comparison isn't capturing a specific edge case (e.g. fuzzy match), add a comparison level to capture this case and retrain the model.\n",
+    "\n",
+    "- If the match weight for a comparison is looking unusual, refer to the [`match_weights_chart`](./match_weights_chart.ipynb) to see the match weight in context with the rest of the comparison levels within that comparison. If it is still looking unusual, you can dig deeper with the [`parameter_estimate_comparisons_chart`](./parameter_estimate_comparisons_chart.ipynb) to see if the model training runs are consistent. If there is a lot of variation between model training sessions, this can suggest some instability in the model. In this case, try some different model training rules and/or comparison levels.\n",
+    "\n",
+    "- If the \"Prior\" match weight is too small or large compared to the match weight provided by the comparisons, try some different determininstic rules and recall inputs to the [`estimate_probability_two_records_match` function](../linker.md#splink.linker.Linker.estimate_probability_two_random_records_match).\n",
+    "\n",
+    "- If you are working with a model with term frequency adjustments and want to dig deeper into the impact of term frequency on the model as a whole (i.e. not just for a single pairwise comparison), check out the [`tf_adjustment_chart`](./tf_adjustment_chart.ipynb).\n"
```

Comment:
If we add a "model refinement" guide somewhere into the docs at some stage, it likely makes sense to add a link to here.